WcaOnRails/config/locales/en.yml

```diff
@@ -1077,7 +1082,8 @@ en:
       title: "My competitions"
       disclaimer: "Only competitions that use the WCA registration system are shown here. If your competition isn't listed here, check the competition website's own competitor list."
       past_competitions: "Past competitions"
-    #context: on the competition's information page
```

Comment:
This shouldn't get removed, just moved below.

Reply:
Whoops, done.